yaml.properties

```diff
@@ -1,15 +1,21 @@
 category=Languages
 description=Allows the analysis of YAML documents
 homepageUrl=https://github.com/sbaudoin/sonar-yaml
-archivedVersions=1.5.2,1.6.0
-publicVersions=1.7.0
+archivedVersions=1.5.2,1.6.0,1.7.0
+publicVersions=1.8.0
 
 defaults.mavenGroupId=com.github.sbaudoin
 defaults.mavenArtifactId=sonar-yaml-plugin
 
 
+1.8.0.description=Brings support for SonarQube 9.4+ and 10; adds a rule and improves some others; fixes some issues
```

Comment:
9.4 is pretty old at this point. Could you maybe talk about 9.9 instead, please?

Reply:
Why not. The thing is that 1.7 is known not to work with SQ 9.4 and beyond but version 1.8 works with SQ 9.4 and beyond. I just wanted to be as sharp as possible in the versions and allow the SQ 9.x users to be able to use the latest available version as it now works as of SQ 9.4.

Still want to mention 9.9 only?